src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -655,45 +663,111 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 				});
 				widget.ariaLabel = ariaLabel;
 				widget.tabIndex = 0;
+				this.attachButtonAndDisposables(widget, index, attachment);
+			} else if (attachment.isImage) {
+				let buffer = new Uint8Array(0);
+				const ariaLabel = localize('chat.imageAttachment', "Attached image, {0}", attachment.name);
```

Comment:
I guess you didn't start this trend, but we shouldn't be copying/pasting code between the two places that attachments are rendered

Reply:
Can you try to reduce the duplication though? I think the image attachments are a bit more complex and it's just a lot of code to copy.

Reply:
Feel free to do that separately after merging this though.